contracts/solidity/script/Upgrade.s.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
```

Comment:
why is it 0.8.19? we were using 0.8.21

Reply:
The change of version may have become outdated since I was experimenting with different things. I'll revert everything to the version it was when I initially set it up (`0.8.21`)